src/shadowbox/docker/run_action.sh

```diff
@@ -14,19 +14,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-touch /tmp/config.json
-source $ROOT_DIR/src/shadowbox/scripts/make_certificate.sh
+do_action shadowbox/docker/build
+
+OUTLINE_DIR=/tmp/outline
+touch "$OUTLINE_DIR/config.json"
+source $ROOT_DIR/src/shadowbox/scripts/make_test_certificate.sh "${OUTLINE_DIR}"
 
 # TODO: mount a folder rather than individual files.
 declare -a docker_bindings=(
-  -v /tmp/config.json:/root/shadowbox/shadowbox_config.json
-  -v /tmp/stats.json:/root/shadowbox/shadowbox_stats.json
+  -v "$OUTLINE_DIR/config.json":/root/shadowbox/shadowbox_config.json
+  -v "$OUTLINE_DIR/stats.json":/root/shadowbox/shadowbox_stats.json
   -v ${SB_CERTIFICATE_FILE}:${SB_CERTIFICATE_FILE}
   -v ${SB_PRIVATE_KEY_FILE}:${SB_PRIVATE_KEY_FILE}
   -e "LOG_LEVEL=${LOG_LEVEL:-debug}"
   -e SB_API_PREFIX=TestApiPrefix
-  -e SB_CERTIFICATE_FILE
-  -e SB_PRIVATE_KEY_FILE
+  -e SB_CERTIFICATE_FILE=${SB_CERTIFICATE_FILE}
+  -e SB_PRIVATE_KEY_FILE=${SB_PRIVATE_KEY_FILE}
 )
 export DOCKER_CONTENT_TRUST=${DOCKER_CONTENT_TRUST:-1}
-docker run --rm -it --network=host --name shadowbox "${docker_bindings[@]}" outline/shadowbox
+sudo docker run --rm -it --network=host --name shadowbox "${docker_bindings[@]}" outline/shadowbox
```

Comment:
From the conversations with Cure53 and other resources, it seems the proper setup for docker is to run as root, since it can allow privilege escalation for regular users. So using `sudo` seems to be the recommended way.

Reply:
The Docker daemon has to run as root - by default only root can speak with the daemon but it's straightforward to grant access to regular users. I definitely don't want to have to give my password for every call to Docker!

Reply:
Adding the user to the docker group was one of the vulnerabilities pointed out by Cure53 (https://github.com/Jigsaw-Code/outline-cure53/issues/2). It's against security best-practices to do that. 

Once you sudo in a process, you don't need to do it again. You won't need to keep typing the password.
Adding the user

Reply:
The document you are linking also tells about it:
![image](https://user-images.githubusercontent.com/113565/50993943-dc26a880-14e8-11e9-8462-e1d637fb8823.png)

Reply:
Removed sudo, and fixed the issue with content trust.

Reply:
Awesome, thank you.